pyccel/codegen/printing/ccode.py

```diff
@@ -2353,6 +2362,13 @@ def _print_PythonImag(self, expr):
     def _print_PythonConjugate(self, expr):
         return 'conj({})'.format(self._print(expr.internal_var))
 
+    def _print_ListAppend(self, expr):
```

Comment:
Append should be mentioned in the changelog. This probably also fixes extend

Reply:
Don't forget to activate the associated tests. Maybe this tiny function should be in its own PR ?

Reply:
done #1982, however the tests require C wrapping for lists as returns of functions as well as the ability to print lists

Reply:
Ah. So it's either blocked by the associated issue or the returns need changing to a len call and elements and then it's blocked by Farouk's issue